### ISSUE TITLE
mongodb connector

### ISSUE THREAD

**Issue by fabulousduck** [Kube-f/kit#5]
Currently there is no connector present. It would save a lot of work having already done this before starting a new project. this will be preferably be done outside of the `services` folder in something like a modules folder.

**Comment by fabulousduck** [comment]
added in 2ec0ff322e8db5fece23f63c5fdc0a4b00f1c7d6. closing this.
